src/main/java/com/bookbla/americano/domain/member/repository/MemberPostcardRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.bookbla.americano.domain.member.repository;
+
+import com.bookbla.americano.domain.member.repository.entity.MemberPostcard;
+import feign.Param;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.Optional;
+
+public interface MemberPostcardRepository extends JpaRepository<MemberPostcard, Long> {
+    Optional<MemberPostcard> findMemberPostcardByMember_Id(@Param("memberId") Long memberId);
+
+    @Modifying
```

Comment:
```suggestion
    @Modifying(clearAutomatically = true)
```

업뎃 된 후 영속성 컨텍스트에 반영해주면 좋을 것 같습니다!

Reply:
수정했습니다. 피드백 감사합니다!:)